### DISCUSSION TITLE
How to remove semicolons

### DISCUSSION THREAD

**Discussion by nailuox** [antfu/eslint-config#522] [Q&A]
Some components do not want to be automatically imported. When saving automatically, there will be a semicolon after the manually imported statement. I have made some configurations, but the semicolon is still present. How can I remove the semicolon?

Here is the configuration of eslint.config.js: 
![image](https://github.com/antfu/eslint-config/assets/49058926/4028a76f-92e1-40d5-afea-0d64395a5e24)

![image](https://github.com/antfu/eslint-config/assets/49058926/1ee8971d-9cc9-493a-a7ff-3007eaf42c20)

**Comment by Dimava** [comment]
1. Press CtrlShiftP - FormatWith and make sure the default is ESLint. 
2. Check that https://github.com/antfu/eslint-config#vs-code-support-auto-fix-on-save stuff is configured correctly
